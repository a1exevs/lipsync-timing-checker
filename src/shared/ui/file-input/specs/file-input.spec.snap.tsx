import { render } from '@testing-library/react';
import React from 'react';

import FileInput from 'src/shared/ui/file-input/file-input';

describe('FileInput snapshots', () => {
  it('renders default', () => {
    const { asFragment } = render(<FileInput accept=".txt" handleFileUpload={jest.fn()} dataTestId="file-input" />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders with different accept', () => {
    const { asFragment } = render(
      <FileInput accept=".jpg,.png" handleFileUpload={jest.fn()} dataTestId="file-input" />,
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders with no dataTestId', () => {
    const { asFragment } = render(<FileInput accept=".txt" handleFileUpload={jest.fn()} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
