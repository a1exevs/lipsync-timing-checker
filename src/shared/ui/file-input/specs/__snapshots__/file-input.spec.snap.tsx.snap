// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`FileInput snapshots renders default 1`] = `
<DocumentFragment>
  <input
    accept=".txt"
    data-testid="file-input"
    style="display: none;"
    type="file"
  />
</DocumentFragment>
`;

exports[`FileInput snapshots renders with different accept 1`] = `
<DocumentFragment>
  <input
    accept=".jpg,.png"
    data-testid="file-input"
    style="display: none;"
    type="file"
  />
</DocumentFragment>
`;

exports[`FileInput snapshots renders with no dataTestId 1`] = `
<DocumentFragment>
  <input
    accept=".txt"
    style="display: none;"
    type="file"
  />
</DocumentFragment>
`;
